crates/http/src/backend/h3/body.rs

```diff
@@ -1,88 +1,40 @@
-#![cfg_attr(not(feature = "quic-quinn"), allow(dead_code))]
-
 use std::future::Future;
 use std::pin::Pin;
 use std::task::{Context, Poll};
 
 use bytes::{Buf, Bytes};
-use h3::quic::{BidiStream, SendStream};
 use h3::server::RequestStream;
 
-#[derive(derive_more::From)]
-pub(crate) enum QuicIncomingBody {
-    #[cfg(feature = "quic-quinn")]
-    Quinn(#[from] QuicIncomingBodyInner<h3_quinn::BidiStream<Bytes>>),
-}
-
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 enum State {
     Data(Option<u64>),
     Trailers,
     Done,
 }
 
-pub(crate) struct QuicIncomingBodyInner<B: BidiStream<Bytes>> {
+pub struct QuicIncomingBody<B: h3::quic::BidiStream<Bytes>> {
```

Comment:
Do we ever use the B generic other then the `B::RecvStream`?  Should we maybe consider removing this bind and passing a recv stream directly rather than the bidi?